content/best_practices.md

```diff
@@ -10,7 +10,7 @@ The following are best practices.
 
 The Quizmaster should:
```

Comment:
caps on "Quizmaster?"  (help me out with the pattern/rule you're following)

Reply:
Punctuation bug. b101002